Mobile/ShopLens/Droid/Source/CameraActivity.cs

```diff
@@ -57,6 +57,10 @@ protected override void OnCreate(Bundle savedInstanceState)
 
             SetContentView(Resource.Layout.Camera);
 
+            //var toolbar = FindViewById<Android.Support.V7.Widget.Toolbar>(Resource.Id.toolbar);
+            //SetSupportActionBar(toolbar);
+            //SupportActionBar.Title = "ShopLens";
```

Comment:
Forgot to remove these comments probably?

Reply:
Fixed.